evm-contracts/src/v0.6/FluxAggregator.sol

```diff
@@ -405,20 +405,23 @@ contract FluxAggregator is AggregatorInterface, AggregatorV3Interface, Owned {
    * @dev Note that for in-progress rounds (i.e. rounds that haven't yet received
    * maxSubmissions) answer and updatedAt may change between queries.
    */
-  function getRoundData(uint256 _roundId)
+  function getRoundData(uint80 _roundId)
     public
     view
     virtual
     override
     returns (
-      uint256 roundId,
+      uint80 roundId,
       int256 answer,
       uint256 startedAt,
       uint256 updatedAt,
-      uint256 answeredInRound
+      uint80 answeredInRound
     )
   {
     Round memory r = rounds[uint32(_roundId)];
+
+    require(r.answeredInRound > 0, "No data present");
```

Comment:
Let's add a comment to the getRoundData method on the Aggregator v3 interface that revert("No data present") is the expected behavior when no data is present

Reply:
Or we could even make it a constant on the interface!